webapp/app/[locale]/stake/_components/tokenRewards.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from 'react'
+
+import { HemiTag } from './rewardTag'
+
+interface Props {
+  rewards: string[]
```

Comment:
Here you could keep the types consistent by importing StakeToken and

```suggestion
  rewards: StakeToken['extensions']['rewards']
```

This way, if you change `rewards` in `StakeToken`, these are kept in sync automatically